dbt/profiles.yml

```
@@ -3,11 +3,19 @@ dbt-civis:
   default: redshift_adapter
   outputs:
     redshift_adapter:
-      type: redshift
+      type: "{{ env_var('DATABASE_TYPE') }}"
       host: "{{ env_var('DATABASE_HOST') }}"
       port: "{{ env_var('DATABASE_PORT') | as_number }}"
       user: "{{ env_var('DATABASE_USERNAME') }}"
       pass: "{{ env_var('DATABASE_PASSWORD') }}"
       dbname: "{{ env_var('DATABASE_NAME') }}"
-      schema: "{{ env_var('DATABASE_SCHEMA', 'default_schema_name') }}"
+      schema: "{{ env_var('DATABASE_SCHEMA') }}"
```

oh looks like we call this `DBT_SCHEMA`
```suggestion
      schema: "{{ env_var('DBT_SCHEMA') }}"
```

https://github.com/civisanalytics/console/blob/9747999200507237c03f9de1d5fca909d54676c6/app/models/job_types/dbt_docker.rb#L46C12-L46C22